src/AppContainer.js

```diff
@@ -1,9 +1,7 @@
 /* eslint-disable global-require */
 
-'use strict';
```

Comment:
I think we need `use strict` here. If ESLint says we don't, it's wrong.

Reply:
We transpile it using Babel, so why do we need to add it?

```sh
$ babel src/AppContainer.js                                                                                                                       
'use strict';

/* eslint-disable global-require */

if (!module.hot || process.env.NODE_ENV === 'production') {
  module.exports = require('./AppContainer.prod');
} else {
  module.exports = require('./AppContainer.dev');
}
```

Reply:
Oh okay.